core/src/com/unciv/models/Tutorial.kt

```diff
@@ -0,0 +1,34 @@
+package com.unciv.models
+
+enum class Tutorial(val value: String) {
```

Comment:
This is a great idea!!!

Reply:
Hm, we don't really reference them in the same way I think, and their information is pretty straightforward
Not sure what problems this would solve, is my point